ui/src/features/get-tests.js

```diff
@@ -26,8 +26,10 @@ import { getColumns } from './configurationColumn';
 import Button from '../components/Button';
 import ErrorDialog from './components/ErrorDialog';
 import _ from 'lodash';
-import { isTestValid } from '../validators/validate-test';
-import { INVALID_TEST_MESSAGE } from '../constants'
+import UiSwitcher from "../components/UiSwitcher";
```

Comment:
It's impossible now edit test in the UI
this is the error:
`Uncaught ReferenceError: isTestValid is not defined
`